### ISSUE TITLE
A Simple Baseline for Semi-supervised Semantic Segmentation with Strong Data Augmentation

### ISSUE THREAD

**Issue by ghuijo** [One-Day-One-Paper/One-Day-One-Paper#267]
* Labeling datasets is very expensive and time-consuming,
  especially in dense prediction problems, such as semantic segmentation
* Semi-supervised learning (SSL)
  * Student uses a large model to generate the pseudo-labels
  * Adding perturbations to the images
  * GAN-based approach
* Propose a framework to apply SSL to semantic segmentation
  * Strong data augmentation (SDA)
  * Distribution-specific batch normalization (DSBN)
  * Self-correction loss (SCL)
  * 모델 구조
  ![image](https://user-images.githubusercontent.com/67627471/185795710-a3ab8da5-4591-42fb-acc2-907d934d51ec.png)
* Distribution mismatch of BN statistics between weak augmentation and strong augmentation
  * Strong augmentation 종류
  ![image](https://user-images.githubusercontent.com/67627471/185795716-6e0eb68e-9956-437d-9310-63804905e155.png)
* Two separate BNs:
  * BNw for samples of weak/standard augmentation
  * BNs for samples of strong augmentation
* BN computes the mini-batch mean (µB) and standard deviation (σB)
* BN uses a learnable scale (γ) and shift (β) parameter
* to transform the normalized distribution
  Moving mean and moving variance are updated
* cross-entropy loss masks the model overfit to noisy labels on some easier classes
  and underlearning on some harder classes in image classification
* In the semi-supervised learning framework, the learning targets are pseudo-labels,
  which may contain unreliable regions
* Self-correction loss
  * assigns an adaptive weight by the confidence of the network output
    to each pixel during the training phase
  * apply a noise-robust term Reverse Cross Entropy to the unreliable regions
* Compared with SOTA methods on the Cityscapes valset
![image](https://user-images.githubusercontent.com/67627471/185795758-3dde1d22-91d4-48ef-8cdf-a6e38f1b80f2.png)
* Qualitative results on the Cityscapes dataset
![image](https://user-images.githubusercontent.com/67627471/185795746-a01b40e9-9602-4f3f-b010-31a1fc531d16.png)
